packages/liveblocks-core/src/protocol/ServerMsg.ts

```diff
@@ -26,6 +26,7 @@ export enum ServerMsgCode {
   COMMENT_DELETED = 404,
   COMMENT_REACTION_ADDED = 405,
   COMMENT_REACTION_REMOVED = 406,
+  THREAD_DELETED = 407,
```

Comment:
We can safely move this a couple lines higher, together with the other `THREAD_` enum members. You don't have to worry about not noticing the code if we add new members later. We already have a lint rule that will fail if there are duplicate members.

Reply:
nice, adjusting :)